KesaApplication/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  KesaApplication
+//
+//  Created by Jiwoo Lim on 2016. 6. 1..
+//  Copyright © 2016년 kesa. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    //Function to display error message
```

Comment:
The comment here is unnecessary since the name of the function tells what it does clearly.